Kickstarter-iOS/Views/Controllers/SettingsAccountViewController.swift

```diff
@@ -122,8 +122,12 @@ final class SettingsAccountViewController: UIViewController, MessageBannerViewCo
   }
 
   private func dismissCurrencyPickerCell() {
+    guard let pickerCell = self.dataSource.removeCurrencyPickerRow() else {
+      return
+    }
+
     tableView.beginUpdates()
-    self.tableView.deleteRows(at: [self.dataSource.removeCurrencyPickerRow()], with: .top)
+    self.tableView.deleteRows(at: [pickerCell], with: .top)
```

Comment:
If you make use of the other suggestion 😄 
```suggestion
    self.tableView.deleteRows(at: [pickerCellIndexPath], with: .top)
```